crates/driver/src/domain/competition/settled/mod.rs

```diff
@@ -0,0 +1,275 @@
+use {
+    super::{
+        auction,
+        order::{self, Side},
+    },
+    crate::{domain::eth, util::conv::u256::U256Ext},
+    number::conversions::big_rational_to_u256,
+};
+
+/// Settlement in an onchain settleable form and semantics, aligned with what
+/// the settlement contract expects.
+#[derive(Debug, Clone)]
+pub struct Settlement {
```

Comment:
can we move this to competition/solutions/scoring.rs and give it a more specific name like score. The struct as it is now doesn't have much to do with an actual settlement, it simply computes the score given net trades and the underlying auction (which is good I believe).

Reply:
Moved.